metaflow/plugins/pypi/micromamba.py

```diff
@@ -277,10 +277,12 @@ def _install_micromamba(installation_location):
     try:
         subprocess.Popen(f"mkdir -p {installation_location}", shell=True).wait()
         # https://mamba.readthedocs.io/en/latest/micromamba-installation.html#manual-installation
+        # requires bzip2
         result = subprocess.Popen(
-            f"curl -Ls https://micro.mamba.pm/api/micromamba/{platform}/latest | tar -xvj -C {installation_location} bin/micromamba",
+            f"curl -Ls https://micro.mamba.pm/api/micromamba/{platform}/latest | tar -xvj -C {installation_location} bin/micromamba >",
```

Comment:
what's with the redirect?

Reply:
good catch - it's spurious. i will remove